app/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/DeclarationXml.scala

```diff
@@ -164,15 +157,15 @@ class DeclarationXml {
     }
   }
 
-  private def maybeElement(elementName: String, maybeElementValue: Option[String]): NodeSeq = {
+  def maybeElement(elementName: String, maybeElementValue: Option[String]): NodeSeq = {
     maybeElementValue match {
       case Some(value) if value.nonEmpty =>
         Elem.apply(null, elementName, scala.xml.Null, scala.xml.TopScope, true, Text(value)) //scalastyle:ignore
       case _ => NodeSeq.Empty
     }
   }
 
-  private def maybeInvoiceLine(declaration: Declaration): NodeSeq = {
+  def maybeInvoiceLine(declaration: Declaration): NodeSeq = {
```

Comment:
It doesn't look like this needs to be public.

Reply:
Agree. Will make private.